ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -755,19 +755,22 @@ objects:
           fi
         fi
 
-        if has_variant "ipv6"; then
-          export OPENSHIFT_INSTALL_AZURE_EMULATE_SINGLESTACK_IPV6=true
+        if has_variant "dualstack"; then
+          export OPENSHIFT_INSTALL_EXPERIMENTAL_DUAL_STACK=true
           cat >> ${ARTIFACT_DIR}/installer/install-config.yaml << EOF
         networking:
           networkType: OVNKubernetes
           machineNetwork:
             - cidr: 10.0.0.0/16
             - cidr: fd00::/48
           clusterNetwork:
-            - cidr: fd01::/48
+            - cidr: 10.128.0.0/14
+              hostPrefix: 23
+            - cidr: fd00:10:128::/56
               hostPrefix: 64
           serviceNetwork:
-            - fd02::/112
+            - 172.30.0.0/16
+            - fd00:172:16::/112
```

Comment:
`fd00:172:30` to match the IPv4 one (likewise in the other file)

Reply:
ups, my bad, I always use to match them